opm/output/eclipse/VectorItems/connection.hpp

```diff
@@ -50,7 +50,7 @@ namespace Opm { namespace RestartIO { namespace Helpers { namespace VectorItems
 
             EffectiveKH  =  3, // Effective Kh product of connection
 
-            item12       = 11, // Unknown
+            item12       = 11, // Connection transmissibility factor
```

Comment:
Excellent!  At some point we may wish to rename the item to something more descriptive than `item12`.

Also, as a general point of interest, is this the "raw" transmissibility factor (from cell geometry and permeability data) or is it also affected by productivity index multipliers such as `WPIMULT`?  Otherwise, this **might** be a little redundant given that item 0 is also supposed to contain the connection transmissibility factor.